docs(trtllm): align with upstream-container migration (#9654) (#10074)
ai-dynamo/dynamo: container/README.md

|  /usr/bin/nats-server | COPY from dynamo_base |
|  /usr/local/bin/etcd/ | COPY from dynamo_base |
|  /opt/dynamo/wheelhouse/ | COPY from wheel_builder |
|  upstream Python/site-packages | inherited from the upstream runtime image — `vllm/vllm-openai` (vLLM; multi-arch amd64/arm64, separate tag per CUDA family), `lmsysorg/sglang` (SGLang; multi-arch amd64/arm64), or `nvcr.io/nvidia/tensorrt-llm/release` (TRT-LLM; multi-arch amd64/arm64) |

> |  /workspace/ | COPY from build context. Subset of the source tree, varies by framework. Common to all three: `tests`, `examples`, `dev`, `components/src/dynamo/{common,frontend,<framework>}`. vLLM and TRT-LLM also copy `lib`; SGLang and TRT-LLM also copy `deploy` and `components/src/dynamo/mocker`; SGLang additionally copies `recipes`. See each framework's `templates/<framework>_runtime.Dockerfile` for the exact list. |

Reviewer: Readability nit (optional): this cell is now ~5 lines of prose inside a table row, which makes the rest of the column hard to scan. Content is correct as-is; if you want to keep the detail without bloating the table, one option is to leave the cell as `COPY from build context (subset varies by framework — see below)` and add a short list/footnote directly under the table.

```suggestion
|  /workspace/ | COPY from build context — subset varies by framework (see note below) |
```

Then a 3-line bullet list under the table:
- Common to all three frameworks: `tests`, `examples`, `dev`, `components/src/dynamo/{common,frontend,<framework>}`
- vLLM and TRT-LLM additionally copy `lib`; SGLang and TRT-LLM additionally copy `deploy` and `components/src/dynamo/mocker`; SGLang additionally copies `recipes`.
- See each framework's `templates/<framework>_runtime.Dockerfile` for the exact list.

Purely formatting — fine to skip.
